src/variants/model/VariantsSearchRequestByVendor.ts

```diff
@@ -0,0 +1,7 @@
+import VariantsSearchRequestByBusinessId from './VariantsSearchRequestByBusinessId';
+import PaginationQueryRequest from '../../model/PaginationQueryRequest';
+
+export default interface VariantsSearchByVendor extends VariantsSearchRequestByBusinessId, PaginationQueryRequest {
```

Comment:
You are missing https://docs.channelape.io/#d8bf9ddf-c384-4cf6-b3d9-c9f696c40e11 (variant search by tag). This is a recent additional within the past few months so it probably wasn't in your original SDK.

Please add a `VariantSearchByTag` interface and hook it up accordingly.